### ISSUE TITLE
Ürün Fotolarının Kolayca Girilebilmesi

### ISSUE THREAD

**Issue by GoogleCodeExporter** [ahmet328/sambapos#32]
```
Menü Listesi içinde yer alan Ürün Fotograflarının kolay bi yöntemle 
girilebilmesini sağlayacak bir yapı kuralım.

```

Original issue reported on code.google.com by `mehmetak...@gmail.com` on 23 Sep 2011 at 1:17

**Comment by GoogleCodeExporter** [comment]
```
Ürtün fotografları diskte bulunan yerini tam olarak yazmak yerine yanına 
bir Browse tuşu koyalım ve fotografların olduğu yeri bilgisayarın içinden 
seçelim.

Farklı dosya uzantıları ve farklı yazım biçimleri ve yazım hataları da 
bu şekilde giderilşmiş olur.
```

Original comment by `mehmetak...@gmail.com` on 23 Sep 2011 at 1:34

**Comment by GoogleCodeExporter** [comment]
```
Resim dosyalarının belirlenmesi ve otomatik seçimi için "Convention Based" 
bir otomatik tanımlama yöntemi geliştirilebilir. Önceden tanımlanan 
c:\images klasöründe kola.jpg diye bir resim varsa program hiç tanımlama 
yapılmadan otomatik olarak kola düğmesine bu resmi yükleyebilir.
```

Original comment by `emree...@gmail.com` on 24 Sep 2011 at 2:48
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Resim kullanmak istemeyen kullanıcılarda bu özelliği kaldırmak için teker 
teker yolların silinmesi gerekliliği doğabilir.
Bunun yerine kullanmak isteyen kullanıcılar için resimlerin yolunu 
yazmaktansa SEÇ tuşu daha kullanışlı olabilir.
```

Original comment by `mehmetak...@gmail.com` on 24 Sep 2011 at 9:45

**Comment by GoogleCodeExporter** [comment]
```
Tamam 2. yorumu görmezden gelelim. Konuyu "browse" ile çözüyoruz.
```

Original comment by `emree...@gmail.com` on 24 Sep 2011 at 12:59

**Comment by GoogleCodeExporter** [comment]
Original comment by `emree...@gmail.com` on 25 Sep 2011 at 1:38
- Added labels: **Milestone-Release-2.5x**

**Comment by GoogleCodeExporter** [comment]
Original comment by `emree...@gmail.com` on 25 Sep 2011 at 1:38

**Comment by GoogleCodeExporter** [comment]
Original comment by `emree...@gmail.com` on 29 Sep 2011 at 11:33
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision 5677c3e113f3.
```

Original comment by `emree...@gmail.com` on 29 Sep 2011 at 3:13
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
... eklenerek çözüldü
```

Original comment by `mehmetak...@gmail.com` on 4 Oct 2011 at 11:57
- Changed state: **Verified**
